gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 Google Inc. All Rights Reserved.
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Ah see, all I know is that I copy/pasted a "(C) 2015" notice onto the files. Since I didn't want to backdate any file I changed it to 2016. Surely those files are still copyrighted today so that message should be fine?

Let me know if you want me to change JobStatistics.java's text back.